### ISSUE TITLE
Backbutton event is not working with Quasar 2.0.9 version

### ISSUE THREAD

**Issue by anandtripathi5** [quasarframework/quasar#7827]
**Describe the bug**
Backbutton event is not working for quasar version 2.0.9 version with cordova version 9.0.0. 

I am using the below code

```
document.addEventListener(
      "backbutton",
      e => {
        console.log("backbutton event logged")
      },
      false
);
```

There is no log inside the back button but it was working for older version

**Comment by rstoenescu** [comment]
Hi. Backbutton is described in [the docs](https://quasar.dev/vue-directives/go-back#Cordova%2FCapacitor). Just disable it for Cordova then you can handle it yourself.

**Comment by anandtripathi5** [comment]
I already went through the docs, after that, I raised this issue.
```
framework: {
      iconSet: "material-icons", // Quasar icon set
      lang: "en-us", // Quasar language pack
      config: {
        screen: {
          bodyClasses: true // <<< add this
        },
        cordova: {
          backButtonExit: false
        },
        capacitor: {
          // Quasar handles app exit on mobile phone back button
          // Requires Quasar v1.9.3+ for true/false, v1.12.6+ for '*' wildcard and array values
          backButtonExit: false
        }
      },
}
```

**Comment by anandtripathi5** [comment]
Any update on this!

**Comment by rstoenescu** [comment]
Cordova has always been a pain in this regard. You can only register one callback for the 'backbutton' (which Quasar gets to do first). Then any subsequent callbacks are discarded. Will have to add a feature to register your own handler (and by doing so to completely disable Quasar's). Not really recommended for UX purposes, but at least you'll have the option.
May I ask why do you need this, though? Maybe what you want can be done in some other elegant way.

**Comment by anandtripathi5** [comment]
Yeah, I needed that, cause if you know how Kotlin or regular app development works is. We have a backbutton function for every page or activity/fragment. So in the case of Quasar also I want to add functionality so that:

If I'm on x page that is a homepage, then on the backbutton it can exit by giving a warning
if I'm on y page then maybe on backbutton I want to go to some w page
and on w page, I want to go to the home page. 

So basically I don't want to follow the stack of Vue router instead want my custom functionality of the back button. In previous version of the quasar, I have written my custom event handler that was handling all this but in the 2.0.9 version of Quasar, the backbutton handler event is not registered.

**Comment by rstoenescu** [comment]
Please make the distinction between "@quasar/app" version (the local CLI) and "quasar" package (the UI). You are reporting a UI issue. Will add the ability to completely disable Quasar's backbutton handling.

**Comment by rstoenescu** [comment]
quasar.conf.js > cordova > backButton: false will do the trick for you.
Will be available in Quasar v1.14.1.

**Comment by anandtripathi5** [comment]
I guess the merge will solve the problem. What is the possible ETA of v1.14.1 of quasar UI!

**Comment by rstoenescu** [comment]
Today @anandtripathi5

**Comment by anandtripathi5** [comment]
Thanks @rstoenescu it's working now. Thanks for the quick solution.

**Comment by SahilKindarle** [comment]
> Yeah, I needed that, cause if you know how Kotlin or regular app development works is. We have a backbutton function for every page or activity/fragment. So in the case of Quasar also I want to add functionality so that:
> 
> If I'm on x page that is a homepage, then on the backbutton it can exit by giving a warning
> if I'm on y page then maybe on backbutton I want to go to some w page
> and on w page, I want to go to the home page.
> 
> So basically I don't want to follow the stack of Vue router instead want my custom functionality of the back button. In previous version of the quasar, I have written my custom event handler that was handling all this but in the 2.0.9 version of Quasar, the backbutton handler event is not registered.

@anandtripathi5 can you paste a snippet of such example here

I'm trying to implement it in this way
  document.addEventListener(
      "backbutton",
      () => {
        // evt.preventDefault()
        if (this.$route.path == "/apps/iron") {
          this.$q.notify("back button from iron");
          this.$router.replace("/home")
         }
        },false)

**Comment by anandtripathi5** [comment]
Hi @SahilKindarle I have used the backButton property of Cordova in quasar.conf.js
![image](https://user-images.githubusercontent.com/10723482/107844847-03a17b00-6dfd-11eb-93f9-12a7bf6f3c79.png)

This resolved the problem

**Comment by fe-sak** [comment]
I've added cordova.backButton = false in quasar.conf.js file and added my event listener for the 'backbutton' event.
The event object is properly passed to my event listener callback, but I cannot override the default behaviour (using e.preventDefault).
So, it is not possible to write custom behaviour for the back button in my case, which is a problem
quasar 2.0.0
cordova-android 10.1.1

**Comment by rstoenescu** [comment]
@fe-sak Your problem has to do more with Cordova than with Quasar. If you've configured backButton to be `false` then Quasar UI does not touch the event.

**Comment by fe-sak** [comment]
I see. Thanks for answering!
